packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -157,6 +164,9 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
             {
                 [CoreClasses.ACTIVE]: this.state.isInputFocused,
             },
+            {
+                [CoreClasses.DISABLED]: this.props.disabled,
```

Comment:
Put this in the same object that was already in place:

```tsx
{
    [CoreClasses.ACTIVE]: this.state.isInputFocused,
    [CoreClasses.DISABLED]: this.props.disabled,
}
```

Reply:
@cmslewis ah yes, I didn't realize you could do that.

Reply:
Nah, it just enforces a global max of 120 chars. To achieve 80-char lines in comments, I use the VS Code `Rewrap` plugin: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap